### ISSUE TITLE
xiFDR CSV output doesn't match xiSPEC/xiVIEW CSV input

### ISSUE THREAD

**Issue by colin-combe** [Rappsilber-Laboratory/xiFDR#33]
Would be good to synch up xiFDR CSV output and xiSPEC/xiVIEW input CSV. Differences are:

CSM level ouput:
- in xiFDR output column is called "exp m/z"; in xiSPEC/xiVIEw input its called "ExpMz"
- there is no "CalcMz" in xiFDR output but xiSPEC/xiVIEW expect it

Link level output:
- "fromSite" in xiFDR CSV is called "AbsPos1" in xiVIEW CSV input
- "toSite" in xiFDR CSV is called "AbsPos2" in xiVIEW CSV input

**Comment by lutzfischer** [comment]
I could see one solution that does not affect anything using either file at the moment - and that would be to accept alternative input names for xiVIEW csv-parser.  Alternative is to break some things in one of the two sites (or software that outputs  xiVIEW or software that uses xiFDR output). The less problematic "breaking" point is to adapt xiFDR output - as there mainly scripts in our lab would be affected.

**Comment by colin-combe** [comment]
> accept alternative input names for xiVIEW csv-parser. 
yeah, it might be the best solution, but the python 2 version of the parser that the xiview.org website currently uses is completely unmaintained.

**Comment by colin-combe** [comment]
newer versions of xiFDR have an export option for xiVIEW csv files
